core/src/lib/internal/net/Response.ts

```diff
@@ -1,7 +1,7 @@
 interface Response {
```

Comment:
re: my other comments about the header stuff, this should probably be a class with a constructor that preprocesses the headers to make them all have lowercased keys.